doc/tutorials/cf.md

```diff
@@ -255,7 +255,7 @@ extern size_t rank;
 // Build the CF object and perform the decomposition.
 // The constructor takes a default-constructed factorizer, which, by default,
 // is of type NMFALSFactorizer.
-CF cf(data, NMFALSFactorizer(), neighborhood, rank);
+CFType cf(data, NMFPolicy(), neighborhood, rank);
```

Comment:
I thought that `CF` was typedef'ed to `CFType<>` (the default), but it seems like that code is not actually there in `src/mlpack/methods/cf/cf.hpp`!  Do you want to instead add that typedef, and then this change shouldn't be needed?

Reply:
Yes, I had added that typedef.